automation/tincrepo/main/pxf/features/hive/hive_partitioned_table_orc_acid/runTest.py

```diff
@@ -0,0 +1,12 @@
+from mpp.models import SQLTestCase
+from mpp.models import SQLConcurrencyTestCase
+
+class PxfHivePartitionedTableOneFormat(SQLConcurrencyTestCase):
```

Comment:
```suggestion
class PxfHivePartitionedACIDTableOneFormat(SQLConcurrencyTestCase):
```
also why `OneFormat` ?